### PR TITLE
Make bash completions work on a Mac

### PR DESCRIPTION
**Pull request by eparis**

### CODE REVIEW

**Comment by eparis** [comment]
No, the help flags are not actually disappearing because of this update to cobra, just their appearance in the docs....

**Comment by eparis** [comment]
ok to test

**Comment by quinton-hoole** [comment]
Thanks @eparis.  Could you split the spf13 upgrade into a separate PR please?

**Comment by eparis** [comment]
Not really. All this commit does it update spf13/cobra and regen docs. Those need to be semi-atomic.

**Comment by quinton-hoole** [comment]
Fair enough.  Needs a rebase and merge conflict resolution.  Then LGTM.

**Comment by eparis** [comment]
updated. Thanks!

File: docs/user-guide/kubectl/kubectl_scale.md [hunk 1]
```diff
@@ -109,7 +108,7 @@ $ kubectl scale --replicas=5 rc/foo rc/bar
 
 * [kubectl](kubectl.md)	 - kubectl controls the Kubernetes cluster manager
 
-###### Auto generated by spf13/cobra at 2015-08-21 17:18:05.165785015 +0000 UTC
+###### Auto generated by spf13/cobra at 2015-09-03 21:06:22.471116954 +0000 UTC
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
```
Comment (Mobomo5) [inline docs/user-guide/kubectl/kubectl_scale.md]: O

File: docs/user-guide/kubectl/kubectl_version.md [hunk 2]
```diff
@@ -83,7 +82,7 @@ kubectl version
 
 * [kubectl](kubectl.md)	 - kubectl controls the Kubernetes cluster manager
 
-###### Auto generated by spf13/cobra at 2015-08-21 17:18:05.1688832 +0000 UTC
+###### Auto generated by spf13/cobra at 2015-09-03 21:06:22.476464324 +0000 UTC
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
 [![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/user-guide/kubectl/kubectl_version.md?pixel)]()
```
Comment (gregorynicholas) [inline docs/user-guide/kubectl/kubectl_version.md]: what is this ga analytics beacon tracker & badge ?!??  bad-ass.  what (if any) open-source project does it stem from?

**Pull request merged**